### DISCUSSION TITLE
FritzOS Webgui broken in 7581

### DISCUSSION THREAD

**Discussion by Tone866** [Freetz-NG/freetz-ng#1061] [Allgemein]
I compiled an image with the same settings for a 7490 and a 7581 (7.18).
The 7490 works fine, but there are many settings in Fritzos on the 7581 that are broken and can't changed.
E.g.: SIP, Internetanbieter, WLAN doesn't work and the dashboard is not displayed at all.

This needs to be fixed. What logs could help to debug this?

**Comment by ableeker** [comment]
Same here.
I've built firmware dated 27th of October. I've flashed this firmware successfully, and the 7851 appears to be working, the internet connection, wireless, and phones are functioning. But the AVM web interface in a number of places are not working correctly, the Overview for instance only shows a header Interfaces, and the rest of the page is blank, and there aren't Telephone Devices, or Telephone Numbers. However, a number of pages are displayed correctly.
The following pagers have issues:
Overview: empty.
Internet Connection: not showing my internet provider, nor is it available in the list.
Answering Machine: seems to be empty.
Telephone Devices: seems to be empty.
Telephone Numbers: seems to be empty.
Cordless Telephones: seems to be empty.
Security: seems to be empty.
Wizards: seems to be empty.
There may be others that aren't working correctly.

**Comment by fda77** [comment]
Disable everything, first the remove+other patches and search for the cause

**Comment by ableeker** [comment]
These are the default settings for `make menuconfig`. Should I disable the following?
Remove:
Remove leftover
Remove module bins
Remove dtrace
Other:
Patch webif features
Add udevmount
Custom /etc/services

**Reply by fda77** [reply]
Until you found the source of your problem.
As this device is not taged UNTESTED it worked for someone else
